tests/HasMetaTest.php

```diff
@@ -9,6 +9,17 @@ public function testUserHasMeta()
         $post = Post::published()->hasMeta('username', 'juniorgrossi')
             ->first();
 
+        $posts = Post::published()->hasMeta('city', ['amsterdam', 'london'])
```

Comment:
Hey @lasmanis did you changed the testing database to include the `city` taxonomy and `amsterdam` and `londom` terms?